### PR TITLE
<PageSection/>- new component

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
New component

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6686

File: packages/wix-style-react/src/PageSection/docs/index.story.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }),
+
+    tabs([
+      tab({
+        title: 'Description',
+        sections: [
+          description({
+            title: 'Description',
+            text:
+              'This line here should briefly describe component in just a sentence or two. It should be short and easy to read.',
```
Comment (TalZa1810) [inline packages/wix-style-react/src/PageSection/docs/index.story.js:49]: @justinara 
Can you please send me the component descrition?

File: packages/wix-style-react/src/PageSection/PageSection.st.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+:import {
+  -st-from: "../Foundation/stylable/colors.st.css";
+  -st-named: G30,R30;
+}
+
+.root {
+  -st-states: showDivider;
+  padding-block-start: 24px;
+  /*padding-inline-start: 48px;*/
```
Comment (TalZa1810) [inline packages/wix-style-react/src/PageSection/PageSection.st.css:9]: The vertical paddings are part of the `<Page.Content/>` component so they are not needed here
@justinara

File: packages/wix-style-react/src/Page/Page.js [hunk 3]
```diff
@@ -601,7 +605,7 @@ Page.propTypes = {
    * */
   scrollProps: PropTypes.shape(ScrollableContainerCommonProps),
 
-  /** Accepts these components as children: `Page.Header`, `Page.Tail`, `Page.Content`, `Page.FixedContent`. Order is insignificant. */
+  /** Accepts these components as children: `Page.Header`, `Page.Section`, `Page.Tail`, `Page.Content`, `Page.FixedContent`. Order is insignificant. */
```
Comment (TalZa1810) [inline packages/wix-style-react/src/Page/Page.js:608]: The page section is a child component of the page content, so I wasn't sure whether to mention this here or not.

File: packages/wix-style-react/src/PageSection/docs/index.story.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+          description({
+            title: 'Description',
+            text:
+              'PageSection is a standalone element that divides content of a page into multiple sections.',
+          }),
+
+          description({
+            text:
+              'Use it to:\n' +
+              '- Divide content into sections on a page.\n' +
```
Comment (TalZa1810) [inline packages/wix-style-react/src/PageSection/docs/index.story.js:66]: When using bullets, the text size is a bit bigger than without bullets. This is how storybook utils works. FYI
@justinara

File: packages/wix-style-react/src/Page/Page.js (hunk 3 above)
Comment (mykas) [reply packages/wix-style-react/src/Page/Page.js:608]: I believe it is important to not mention here if Page.Section is specifically meant for Page.Content.

File: packages/wix-style-react/src/PageSection/PageSection.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Heading from '../Heading';
+import Text from '../Text';
+import Divider from '../Divider';
+import { st, classes } from './PageSection.st.css';
+import { dataHooks } from './constants';
+
+/** Page Section */
+class PageSection extends React.PureComponent {
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/PageSection.js:10]: I would suggest converting this class already to just function. I know we did not decide yet on general use case, but more and more components are coming as function rather classses.

**Review by justinara — CHANGES_REQUESTED** [review]
- [x] missing 12px bottom padding for a content (bottom divider is too close to title)

**Comment by TalZa1810** [comment]
> * [ ]  missing 12px bottom padding for a content (bottom divider is too close to title)

I did that but it was not published to storybook for some reason. I'll try to dummy commit this so it is published to storybook as well.

File: packages/wix-style-react/src/PageSection/test/PageSection.visual.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  title: 'title',
+};
+
+const tests = [
+  {
+    describe: '',
+    its: [
+      {
+        it: 'title',
+        props: {},
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/test/PageSection.visual.js:23]: Shouldn't props here include some title? Or you rely on commonProps?

File: packages/wix-style-react/src/PageSection/PageSection.uni.driver.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import { baseUniDriverFactory, findByHook } from '../../test/utils/unidriver';
+import { dataHooks } from './constants';
+
+export const pageSectionDriverFactory = (base, body) => {
+  return {
+    ...baseUniDriverFactory(base, body),
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/PageSection.uni.driver.js:6]: I believe our users would like to have methods like these:
```
titleExists(),
subtitleExists(),
getTitle(),
getSubtitle()
```

File: packages/wix-style-react/src/PageSection/test/PageSection.visual.js (hunk 6 above)
Comment (TalZa1810) [reply packages/wix-style-react/src/PageSection/test/PageSection.visual.js:23]: I rely on commonProps, but I can add here a different title just to be sure

File: packages/wix-style-react/src/PageSection/docs/examples.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+export const ellipsis = `
+<PageSection
+  title="Track Your Workflows on Simple Boards"
+  subtitle="Manage any work process from start to finish. Track leads, contacts & orders on a simple board."
+  actionsBar={<TextButton>Go to Workflows</TextButton>}
+/>
+`;
+
+export const simpleUsage = `
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/docs/examples.js:17]: @justinara following our new guidelines - do you think it make sense to have such example as the basic example for our users? Maybe it should state that its structure example and then go prop by prop example as separate ones?

File: packages/wix-style-react/src/PageSection/docs/index.story.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+          example({
+            title: 'Divider',
+            text:
+              'Use divider to visually merge elements into one entity when section content is built using grid layout.',
+            source: examples.divider,
+          }),
+
+          example({
+            title: 'Ellipsis',
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/docs/index.story.js:98]: From the title it seems like `Ellipsis` is actually a prop. Maybe different name should be used here for component behaviour under layout restriction. @justinara maybe you have ideas?

File: packages/wix-style-react/src/PageSection/docs/examples.js [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+<PageSection
+  title="Section Title"
+  subtitle="Section subtitle"
+  actionsBar={<TextButton  prefixIcon={<Icons.Add />}>Button</TextButton>}
+/>
+`;
+
+export const commonUseCases = `
+<Page>
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/docs/examples.js:26]: IMO from the visuals I can't understand where these sections are used in. Maybe adding Page Heading might be beneficial to get immediate understanding what this component is about?

File: packages/wix-style-react/src/PageSection/docs/index.story.js [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+      {
+        label: 'Add Item',
+        value: <TextButton prefixIcon={<Add />}>Add Item</TextButton>,
+      },
+    ],
+  },
+
+  sections: [
+    header({
+      sourceUrl: `https://github.com/wix/wix-style-react/tree/master/src/${PageSection.displayName}/`,
```
Comment (mykas) [inline packages/wix-style-react/src/PageSection/docs/index.story.js:50]: Missing component in header representation.

File: packages/wix-style-react/src/PageSection/docs/index.story.js [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+          }),
+
+          importExample(),
+
+          divider(),
+
+          title('Examples'),
+
+          example({
+            title: 'Simple Usage',
```
Comment (TalZa1810) [inline packages/wix-style-react/src/PageSection/docs/index.story.js:83]:
```suggestion
            title: 'Structure',
```

**Review by justinara — CHANGES_REQUESTED** [review]
I missed one tiny thing:
- [ ] Subtitle font colour should be D20

**Review by justinara — APPROVED** [review]

**Pull request merged**